exo-go/src/cmd/shared.go

```
@@ -11,3 +19,22 @@ func printHelpIfNecessary(cmd *cobra.Command, args []string) bool {
 	}
 	return false
 }
+
+func getAppConfig() types.AppConfig {
+	appDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	appConfig, err := appConfigHelpers.GetAppConfig(appDir)
+	if err != nil {
+		log.Fatalf("Cannot read application configuration: %s", err)
```

This function should return an error instead of log.Fatalf or panic